src/main/scala/com/typesafe/sbt/packager/archetypes/TemplateWriter.scala

```diff
@@ -53,6 +53,10 @@ object TemplateWriter {
     sb toString
   }
 
+  private def replaceValues(lines: Seq[String], replacements: Seq[(String, String)], keySurround: String => String): Seq[String] = {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [replaceValues should have a smaller scope by using private[this]](https://www.codacy.com/app/nepomukseiler/sbt-native-packager/file/1559303156/issues/source?bid=2040925&fileBranchId=2489531#l56)